src/main/scala/com/wavesplatform/utx/UtxPoolImpl.scala

```diff
@@ -59,8 +59,9 @@ class UtxPoolImpl(time: Time,
   private[this] object PoolMetrics {
     private[this] val sizeStats  = Kamon.rangeSampler("utx-pool-size", MeasurementUnit.none, Duration.of(500, ChronoUnit.MILLIS))
     private[this] val bytesStats = Kamon.rangeSampler("utx-pool-bytes", MeasurementUnit.information.bytes, Duration.of(500, ChronoUnit.MILLIS))
-    val processingTimeStats      = Kamon.histogram("utx-transaction-processing-time", MeasurementUnit.time.milliseconds)
-    val putRequestStats          = Kamon.counter("utx-pool-put-if-new")
+    val putTimeStats                    = Kamon.histogram("utx-transaction-processing-time", MeasurementUnit.time.milliseconds)
```

Comment:
These should be timers, which is a wrapper around a histogram. It's also a fortunate opportunity to rename all metrics the way we did in matcher (e.g. `node.utx.put-if-new` and `node.utx.pack-unconfirmed`)